modules/rest-api/openapi/openapi.yaml

```diff
@@ -912,6 +1064,15 @@ components:
           - $ref: '#/components/schemas/State'
           - description: Node status.
       description: Node state.
+    OperationResult:
```

Comment:
I think it is enough to use HTTP codes to indicate if something is done or there is an error.